### PR TITLE
Update 020-route-propagation-pt-0.markdown

### PR DESCRIPTION
**Pull request by bjalonmontanes**
I think "there will be a space provided at the bottom of each story for your to record these values" contains a typo, where "your" should be "you"

### CODE REVIEW

File: c2c/070-dataflow-pt-3.markdown [hunk 1]
```diff
@@ -115,7 +115,7 @@ paste it here.
 The source IP is the overlay IP for your app. The comment is the app guid for
 appA. And the mark is...well... where *does* the mark come from?
 
-When a c2c policy is created, the policy server determines if the app has a
+When a c2c policy is created, the policy server determines if the app has aCancel changes
```
Comment (ameowlia) [inline c2c/070-dataflow-pt-3.markdown:118]: Can you fix this?

**Review by ameowlia — DISMISSED** [review]
Everything looks great except one typo. Thank you for making this PR!

**Comment by MarcPaquette** [comment]
Hi @bjalonmontanes, 
Are you able to resolve the comment above? We'd love to get these changed merged in.

Thanks
@MarcPaquette

**Review by MarcPaquette — APPROVED** [review]

File: c2c/070-dataflow-pt-3.markdown (hunk 1 above)
Comment (MarcPaquette) [reply c2c/070-dataflow-pt-3.markdown:118]: Resolving this issue in another PR

**Pull request merged**